gatherling/Data/sql/migrations/57.sql

```diff
@@ -0,0 +1,11 @@
+-- You can't manage a series that doesn't exist
+DELETE FROM series_organizers WHERE series NOT IN (SELECT name FROM series);
+-- Now let's enforce that going forward
+ALTER TABLE series_organizers ADD FOREIGN KEY (series) REFERENCES series(name);
+-- Delete a series that is a typo of a real series (Giovedi Pauper Chanllenges)
+-- and another that never had an event (Heirloom Kaleidoscope).
+-- This is necessary because the system currently doesn't handle a series without a most recent event
```

Comment:
Does this mean that making a new Series is going to crash things?

Reply:
I don't think so, but I'll check.

Reply:
Checked and new series creation is ok after this